packages/discord.js/typings/index.d.ts

```diff
@@ -6878,7 +6877,7 @@ export interface WebhookMessageCreateOptions extends Omit<MessageCreateOptions,
 export interface WebSocketOptions {
   large_threshold?: number;
   version?: number;
-  presence?: GatewayPresenceUpdateData;
+  presence?: PresenceData;
```

Comment:
Why was this changed? The goal here is to have those types being forward compatible and `@discordjs/ws` expects the previous type.

Reply:
I changed it to match the type in the JSDoc. I'll revert this and change it in the JSDoc instead

Reply:
Changing this to the gateway type forces users to pass all of its properties at all times, e.g.:
![image](https://github.com/user-attachments/assets/68aec7f3-bf30-4924-9bc5-2d0067d80c1a)

IMO it doesn't hurt to keep the parsing on our side... Thoughts? @discordjs/core

Reply:
I'm fine with this as long as we do it exactly as we're doing intents (see Qjuh's /ws integration PR).